spec/lucky/link_helpers_spec.cr

```diff
@@ -45,6 +45,14 @@ private class TestPage
       text "Hello"
     end
   end
+
+  def get_route_without_text
+    link to: LinkHelpers::Index
+  end
+
+  def string_path_without_text
+    link to: "/foo"
```

Comment:
I think instead of allowing an empty `link` it should accept a block so you can render images, text, etc. inside of the link

Reply:
Never mind :)